plugins/org.komodo.core/src/config/komodo.cnd

```diff
@@ -56,9 +56,6 @@
  * data sources, and other resources created and being worked on by the user.
  */
 [tko:workspace] > nt:unstructured
-  + tko:repositories (tko:repositories) copy
```

Comment:
Here's my "global workspace" use-case coming to light.. sor ot.  Will users want to change "repos" (working on drastically different VDBs & DSs)  but wish to share Teiid server connections/info and external repo definitions between their persional repos?  If so, then having them in my personal "library" makes sense